### PR TITLE
add 1.11 conformance job on GCE, tweak existing jobs and add to dashboards

### PR DESCRIPTION
**Pull request by BenTheElder**
once this is up we can potentially add this to release blocking as well, and we should consider moving the existing master branch job into the master blocking dashboard.

/area jobs

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -11038,6 +11038,21 @@
       "sig-gcp"
     ]
   },
+  "ci-kubernetes-gce-conformance-latest-1-11": {
+    "args": [
+      "--extract=ci/latest-1.11",
```
[the remark this answers is not in the document]
Comment (krzyzacy) [reply jobs/config.json]: this is fine

**Comment by BenTheElder** [comment]
I think we should add OpenStack to the dashboard(s) well! These have been quite stable and green for a while now cc @dims 

IMO, if any provider can produce stable federated continuous conformance testing for the k/k branches we should use that signal.

**Comment by dims** [comment]
+1 from me @BenTheElder

**Pull request merged**